src/__tests__/Nav.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] Please fix this error as the previous reviewer requested:
![image](https://github.com/mohamedgamaleldin999999/metrics-app/assets/91486791/ade92ff2-db2b-43c5-a99f-99b181a9e5dd)